inc/advanced-navigation.php

```diff
@@ -130,7 +130,7 @@ function form($filter_arg = null) {
 			$filter_taxonomy = $filter_arg['taxonomy'];
 		endif;
 
-		if(isset($filter_arg['depth']) && !empty($filter_arg['depth'])):
+		if(isset($filter_arg['depth']) && !is_null($filter_arg['depth'])):
```

Comment:
@Huyeng **isset** function already checks whether a variable is null. See http://php.net/manual/en/function.isset.php

> isset — Determine if a variable is set and is not NULL

Normally, you want to check if **isset** plus if **!empty** to make sure the variable is properly set

Please adapt..